tests/api/api.spec.ts

```diff
@@ -1145,6 +1145,51 @@ describe("@dataform/api", () => {
         expect(e.message).to.equal("Compilation timed out");
       }
     });
+
+    it("dataform.json is valid", async () => {
+      try {
+        utils.validate(
```

Comment:
i don't think we need to do this `utils.validate()`, we can just call `compile()` and let it throw.

Reply:
Done